static/styles/6.components/component.css

```diff
@@ -1,16 +1,21 @@
 .component {
-    padding: var(--component-padding, 10px);
-    border-radius: var(--borderRadius);
-    box-shadow: var(--boxShadow);
-    background: var(--color-component);
+  padding: var(--component-padding, 10px);
+  border-radius: var(--borderRadius);
+  box-shadow: var(--boxShadow);
+  background: var(--color-component);
+  transition: box-shadow 200ms ease-in-out 500ms;
```

Comment:
You could use the variable for transition-duration: `transition: box-shadow var(--duration-fast) ease-in-out 500ms;`